hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/app.tsx

```diff
@@ -56,48 +57,51 @@ class App extends React.Component<Record<string, object>, IAppState> {
   };
 
   render() {
-    const { collapsed, enableNewUI } = this.state;
+    const { collapsed, enableOldUI } = this.state;
     const layoutClass = classNames('content-layout', { 'sidebar-collapsed': collapsed });
 
 
     return (
       <Router>
         <Layout style={{ minHeight: '100vh' }}>
           {
-            (enableNewUI)
-            ? <NavBarV2 collapsed={collapsed} onCollapse={this.onCollapse} />
-            : <NavBar collapsed={collapsed} onCollapse={this.onCollapse} />
+            (enableOldUI)
+              ? <NavBar collapsed={collapsed} onCollapse={this.onCollapse} />
+              : <NavBarV2 collapsed={collapsed} onCollapse={this.onCollapse} />
           }
           <Layout className={layoutClass}>
             <Header>
               <div style={{ margin: '16px 0', display: 'flex', justifyContent: 'space-between' }}>
-                {(enableNewUI) ? <BreadcrumbsV2 /> : <Breadcrumbs />}
+                {(enableOldUI) ? <Breadcrumbs /> : <BreadcrumbsV2 />}
                 <AntDSwitch
-                  disabled={true}
-                  checkedChildren={<div style={{ paddingLeft: '2px' }}>New UI</div>}
+                  unCheckedChildren={<div style={{ paddingRight: '2px' }}>UI v2</div>}
```

Comment:
Its better to rename labels shown to users as old and new UI as "Switch to Old UI" and "Switch to New UI"

Reply:
Addressed in [9e942d4](https://github.com/apache/ozone/pull/7358/commits/9e942d4368083a49aecc038b9498b18793008f11)
After change:
<img width="1728" alt="Screenshot 2024-10-29 at 12 15 33" src="https://github.com/user-attachments/assets/3de9cb8c-a23c-48c1-9bf6-09125491dc22">